File: app/src/main/java/fr/insapp/insapp/models/Notifications.java
package fr.insapp.insapp.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by thomas on 26/07/2017.
 */

public class Notifications {

    @SerializedName("notifications")
    private List<Notification> notifications;

    public Notifications(List<Notification> notifications) {
        this.notifications = notifications;
    }

    public List<Notification> getNotifications() {
        return notifications;
    }
}

File: app/src/main/java/fr/insapp/insapp/models/Notifications.kt
package fr.insapp.insapp.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by thomas on 26/07/2017.
 * Kotlin rewrite on 30/08/2019.
 */

@Parcelize
data class Notifications(
    @SerializedName("notifications") val notifications: MutableList<Notification>
): Parcelable
